pkg/landscaper/dataobjects/targetlist.go

```diff
@@ -0,0 +1,85 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dataobjects
+
+import (
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+)
+
+// Target is the internal representation of a target.
+type TargetList struct {
+	Targets    []*Target
+	FieldValue *lsv1alpha1.FieldValueDefinition
```

Comment:
the field value is not needed if all values are tracked by the referenced targets?